users_ldap_populate/models/users_ldap.py

```diff
@@ -109,9 +109,14 @@ def get_ldap_entry_dicts(self, conf, user_name='*'):
         conn = self.connect(conf)
         conn.simple_bind_s(conf['ldap_binddn'] or '',
                            conf['ldap_password'] or '')
-        results = conn.search_st(conf['ldap_base'], ldap.SCOPE_SUBTREE,
-                                 ldap_filter.encode('utf8'), None,
-                                 timeout=60)
+        try:
+            results = conn.search_st(
+                conf['ldap_base'], ldap.SCOPE_SUBTREE,
+                ldap_filter.encode('utf8'), None, timeout=60)
+        except Exception:
```

Comment:
`except:` alone should work. Or is it required for the logger?

Reply:
@sebalix A bare except results in a pylint warning. So not required functionally but by code conventions.

Reply:
@NL66278 ok, thanks!